Jack/Property.fs

```diff
@@ -221,28 +343,44 @@ module PropertyBuilder =
         member __.For(m : Property<'a>, k : 'a -> Property<'b>) : Property<'b> =
             Property.bind m k
 
+        member __.For(xs : seq<'a>, k : 'a -> Property<unit>) : Property<unit> =
+            let join m n =
+                Property.bind m (fun _ -> n)
+            let init =
+                Property.success ()
+            xs |> Seq.map k |> Seq.fold join init
```

Comment:
I think this isn't safe yet as we don't dispose the `IEnumerator` unless the whole list is looped over, will fix in a later PR though